packages/cli/src/lib/remix-version-interop.test.ts

```diff
@@ -1,15 +1,18 @@
 import {describe, it, expect} from 'vitest';
 import {
-  convertRouteToV2,
+  convertRouteToV1,
```

Comment:
Interesting this was inverted.

Reply:
Yeah, now the routes are V2 by default so we convert them to V1 if needed.